build/building/multi-platform.md

```diff
@@ -85,13 +90,14 @@ $ docker buildx create --append --name mybuild node-arm64
 $ docker buildx build --platform linux/amd64,linux/arm64 .
```

Comment:
Could add a cross link to https://docs.docker.com/build/ci/github-actions/configure-builder/#append-additional-nodes-to-the-builder for GHA case?